src/server/firefox/ffPage.ts

```diff
@@ -134,11 +134,14 @@ export class FFPage implements PageDelegate {
     if (!frame)
       return;
     const delegate = new FFExecutionContext(this._session, executionContextId);
-    const context = new dom.FrameExecutionContext(delegate, frame);
+    let worldName: types.World|null = null;
     if (auxData.name === UTILITY_WORLD_NAME)
-      frame._contextCreated('utility', context);
-    else if (!auxData.name)
-      frame._contextCreated('main', context);
+      worldName = 'utility';
+    else if (!auxData)
```

Comment:
This used to be `!auxData.name`. Did something change in the protocol?

Reply:
No, I just messed up copy pasting.